.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -25,6 +25,12 @@ If applicable, add screenshots to help explain your problem.
 **Additional context**
 Add any other context about the problem here.
 
+**Notify maintainers**
+<!--
+Please @ people who are in `meta.maintainers` of a package or module.
+If in doubt, check git blame for whomever last touched something.
```

Comment:
Wording suggestion:

```suggestion
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
```

"He/she touched it last!" not "Him/her touched it last!" (is how I remember the difference).